test/elixir_analyzer/exercise_test/check_source_test.exs

```diff
@@ -121,4 +121,89 @@ defmodule ElixirAnalyzer.ExerciseTest.CheckSourceTest do
       """
     ]
   end
+
+  test "full check_source definition in a test file for coverage" do
+    defmodule Coverage do
```

Comment:
This is one of these cases where even though these things are being used somewhere else, somehow the coverage tool doesn't see it. It's a bit messy, but I suppose it doesn't hurt to have redundant tests.